src/core/util.jl

```diff
@@ -66,3 +66,8 @@ align(x,y) = begin
 
   (x,y)
 end
+
+macro sym_str(var)
```

Comment:
This is a trick I just learnt. If we write the macro name as `xxx_str` we can directly call it by `xxx"something"`. In our case, we can create a symbol by `sym"x[1]"`.